tests/in/arraylength.wgsl

```diff
@@ -0,0 +1,12 @@
+[[block]]
```

Comment:
I don't think we want to have a separate test file for each operation.
Would it be possible to make this a part of the `access.wgsl` instead?